docs/howto/profiling.md

```diff
@@ -0,0 +1,156 @@
+# Profiling
+
+Software profiling allows you to inspect the performance of a piece of software, seeing where the bottlenecks are and how they can be improved.
+gperftools is a software profiling package. It contains a CPU profiler, thread-caching malloc library, memory leak detection tool, memory allocation profiler and pprof (discussed later). More information about gperftools and its components can be found on the project's [GitHub](https://github.com/gperftools/gperftools) and [documentation page](https://gperftools.github.io/gperftools/).
+
+## Install gperftools
+
+### Ubuntu
+
+If you're using Ubuntu you can use the following command to install gperftools:
+
+```
+apt install google-perftools libgoogle-perftools-dev
+```
+
+### Other Systems
+
+Alternatively, you can use the following commands to install it:
+
+```
+git clone https://github.com/gperftools/gperftools
+cd gperftools
+git tag -l # checkout the latest release version
+git checkout <gperftools-X.x>
+./autogen.sh
+./configure
+make prefix=/usr libdir=<your/desired/install/dir>
+make prefix=/usr libdir=<your/desired/install/dir> install

```

Comment:
doesn't `prefix=/usr` mean that files apart from library ones are written to `/usr`? That's not writable without root.

Would it work to specify just `prefix=<your/desired/install/dir>`?

Reply:
I've amended the documentation. Tested the changes on Mercury and was able to profile ACTS.